pom.xml

```diff
@@ -4,7 +4,7 @@
 
 	<groupId>com.paywithmybank</groupId>
 	<artifactId>ncl</artifactId>
-	<version>0.4.0</version>
+	<version>0.4.1</version>
```

Comment:
[Warning][Comment]
Be careful to not let your environment (.m2) dirty with wrong versions. Probably it won't happen in that case, but would be nice to work with _-SNAPSHOT_ versions.
Maybe it should be aligned/discussed with _Release Team_?